importer/config/montreal/cfg.json

```diff
@@ -0,0 +1,20 @@
+{
+  "osm_url": "http://download.geofabrik.de/north-america/canada/quebec-latest.osm.pbf",
+  "map_config": {
+    "driving_side": "Right",
+    "bikes_can_use_bus_lanes": true,
+    "inferred_sidewalks": true,
+    "separate_cycleways": false
+  },
+  "onstreet_parking": {
+    "SomeAdditionalWhereNoData": {
```

Comment:
This is fine to start, but it's a tradeoff -- some roads will wind up much wider than they really are, which can exasperate geometry problems. But it's pretty easy to notice and fix -- you can just go explicitly tag street parking there, pull down fresh .osm, and import again.